src/main/java/com/breskul/bibernate/persistence/GenericDao.java

```diff
@@ -84,4 +91,49 @@ private <T> T mapResult(ResultSet resultSet, Class<T> cls) {
           .formatted(cls), e);
     }
   }
+
+  public <T> int executeUpdate(EntityKey<T> entityKey, Object... parameters) {
```

Comment:
Need add this logic:

- After updating it must be changed entity from it
- An entity that we want to update must be present in PersistenceContext otherwise throw an error

Reply:
Dirty checking in the update operation is dependent on the most recent changes in the entity that resides in the persistence context (first-level cache). This raises a question: how the entity we want to update might not be in the PersistenceContext?

Reply:
I suppose this point is not valid